installation-and-configuration/office-365-connection-details.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Office 365 Connection Details
+description: >-
+  This article explains how SysKit Trace connects to Office 365 and collects data.
+author: Vinko Bedek
+date: 3/17/2020
+---
+
+# Office 365 Connection Details
+
+SysKit Trace uses various PowerShell modules to connect to Office 365 and collect data.
+Each of the powershell modules has its own authentication requirements and potential problems.
+
+One of the goals of SysKit Trace is to collect the data in an unattended manner, regularly, with no user interaction required such as signing in. 
+The solution in such cases is to create an Azure Active Directory Application and perform the actions with the identity of the associated service principal.
+Unfortunately, this is not possible for all the workloads because they do not support service principals. Some workloads still require for the data to be collected in the context of a user.  
+
+## Used PowerShell Modules
+SysKit Trace will automatically install the following PowerShell modules to collect the data from Office 365:
+- ReverseDSC
+- SharePointPnPPowerShellOnline
+- MicrosoftTeams
+- AzureAD
+- Microsoft.PowerApps.Administration.PowerShell
+
+In addition to this SysKit Trace uses [Office365DSC](https://github.com/microsoft/Office365DSC) but it is not installed as a module available to the user.
+
+## SysKit Trace Azure AD Application
+When configuring SysKit Trace an Azure Active Directory Application will be created and pre-consented for the user to collect the data from the tenant.
+You can find the SysKit Trace application in the [Azure Portal](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps) once SysKit Trace has been configured. 
+
+There are two ways to authenticate an Azure Active Directory Application with Azure AD when using the applications identity.
+1. application secrets
+2. certificates  
+
+SysKit Trace uses certificates because most of the workloads that are collected support it.
+A certificate is created and imported into the current user personal certificates for both the user configuring SysKit Trace and the service account that is configured to run the SysKit Trace service.
+This is the reason why we also require a specific windows service account for the SysKit Trace windows service. The alternative would be to put the certificate in the local machine certificate store and such a thing would be unsafe on shared machines since the certificate would be available to all users.
```

Comment:
I would exclude this paragraph?

Reply:
to technical or other reasons?

Reply:
Or maybe just exclude part that is explaining alternatives

Reply:
This part where we explain why we need a password every time and what are the alternatives. :)

Reply:
removed the text. better?